packages/flutter/lib/src/cupertino/localizations.dart

```diff
@@ -79,6 +79,17 @@ abstract class CupertinoLocalizations {
   // The global version uses date symbols data from the intl package.
   String datePickerMonth(int monthIndex);
 
+  /// Month that is shown in [CupertinoDatePicker] spinner corresponding to
+  /// the given month index in [CupertinoDatePickerMode.monthYear] mode.
```

Comment:
Can you add another paragraph like this to further explain the linguistics here?  I didn't understand this myself until I read https://github.com/flutter/flutter/issues/130930 carefully!

 > This is distinct from [datePickerMonth] because in some languages, like Russian, the name of a month takes a different form depending on whether it is preceded by a day or whether it stands alone.